tests/Browser/AdminUpdateMenuTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Category;
+use App\Food;
+use App\DailyMenu;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Facades\DB;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+
+class AdminUpdateMenuTest extends DuskTestCase
+{
+    use DatabaseTransactions;
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test show content.
+     *
+     * @return void
+     */
+    public function testContent()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+                ->visit('/daily-menus')
+                ->assertSee("LIST DAILY MENUS");
+        });
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test delete success.
+     *
+     * @return void
+     */
+    public function testUpdateSuccess()
+    {
+        $this->browse(function (Browser $browser) {
+            factory(Category::class, 1)->create()->each(function($c) {
+                $c->foods()->save(factory(Food::class)->make());
+            });
+            factory(DailyMenu::class, 1)->create(['food_id' => 1, 'date' => '2017-09-11']);
```

Comment:
You create data with date = 2017-09-11. If run test after this date, your test will fail. 
You can try to use: `date('Y-m-d', strtotime(' +1 day'))`